content/_releases/22/20.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Product Update - v22.20
+description: Details of the product update v22.20 from 19th of May 2022.
+layout: rel
+version: "22.20"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-05-19
+redirect_from:
+  - /releases/22.20.html
+---
+
+## Improvements and Updates
+
+### Re-authentication for sensitive actions
+
+While working on the platform user interface we try to click our way through
+setup options and there is a chance we might accidentally remove a sensitive data.
+There can be also cases when your account might be compromised and somebody would
+deliberately try to cause harm.
+
+For such cases we introduce re-authentication for sensitive actions. After
+{{page.version}} for the following actions you will be asked to enter your password
+to authenticate:
+
+*   Reading, copying and regenerating your API Keys.
+*   Modifying your user details.
+*   Disabling 2-Factor Authentication (2FA).
+*   Inviting members to contract.
+*   Inviting and removing members from workspaces.
+
+{% include img.html max-width="40%" url="/assets/img/RN/2220/RN2220-reauth.png" title="Sensitive action re-authentication" %}
+
+> **Please Note**: You will be asked to re-authenticate once every 6 hours.
```

Comment:
Suggest to place this directly under the bullets:

**Please note**: Re-authentication will also be required for any sessions longer than 6 hours.